include/wex/GraphicsController.hpp

```diff
@@ -0,0 +1,46 @@
+#include "Common.hpp"
+#include "SFML/Graphics/CircleShape.hpp"
+#include "SFML/Graphics/Drawable.hpp"
+#include "SFML/Graphics/RenderWindow.hpp"
+#include "SFML/System/Vector2.hpp"
+#include "SFML/Window/VideoMode.hpp"
+#include "util.hpp"
+
+namespace wex {
+
+using Circle = sf::CircleShape;
+
+/// \brief A configuration struct for intializing
+/// a wex::GraphicsController.
+struct WindowConfig {
+	u32 winWidth			= 800;
+	u32 winHeight			= 600;
+	double fps				= 60.0;
+	bool vSync				= true;
+	std::string windowTitle = "Wex Game";
+
+	WindowConfig() = default;
```

Comment:
Rule of zero again; also, you probably want fat members on top for optimum padding

Reply:
Wow I never stopped to think that could have made a difference.